app/src/main/kotlin/xyz/duckee/android/MainActivity.kt

```diff
@@ -109,20 +107,47 @@ class MainActivity : ComponentActivity() {
                 }
             }
         }
+
+        // Handle deeplink when the app is opened from a deeplink
+        handleDeeplink(intent)
```

Comment:
Added a call to `handleDeeplink(intent)` in `onCreate()`. This ensures that deeplinks are handled when the app is first opened. According to the Airbridge SDK documentation, it's important to handle deeplinks as early as possible in the app's lifecycle.